frappe/patches/v8_0/update_gender_and_salutation.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2015, Frappe Technologies Pvt. Ltd. and Contributors
+
+from __future__ import unicode_literals
+import frappe
+from frappe.desk.page.setup_wizard.install_fixtures import get_genders_and_salutations
+
+def execute():
+	frappe.db.set_value("DocType", "Contact", "module", "Contacts")
+	frappe.db.set_value("DocType", "Address", "module", "Contacts")
+	frappe.db.set_value("DocType", "Address Template", "module", "Contacts")
+	update_gender_and_salutation()
+	
+def update_gender_and_salutation():
+	records = get_genders_and_salutations()
```

Comment:
`get_genders_and_salutations()` and `add_genders_and_salutations(record)` can be the same function!